src/main.ts

```diff
@@ -521,23 +548,30 @@ async function getAllActionsUsingSearch(
     '+filename:action+language:YAML'
   )
 
+
   for (let index = 0; index < searchResult.length; index++) {
     checkRateLimits(client, isEnterpriseServer)
 
     const fileName = searchResult[index].name
     const filePath = searchResult[index].path
     const repoName = searchResult[index].repository.name
     const repoOwner = searchResult[index].repository.owner.login
-    const isArchived = searchResult[index].repository.archived
```

Comment:
Use this line instead of the extra call above, since the info is already available

![image](https://github.com/devops-actions/load-available-actions/assets/6085745/1718bcc7-2299-4b24-8d43-7bb2a1b6559f)

Reply:
This is not true the searchresult[index].repository only contains a field archived_url but not the archived field. 
That is why I added that extra call.
![image](https://github.com/devops-actions/load-available-actions/assets/119671489/8eb17de2-ff01-4eab-9da7-309db3201a6f)
![image](https://github.com/devops-actions/load-available-actions/assets/119671489/df7e6279-3cdd-40c3-9419-468f77301b4a)

Reply:
Those screenshots seem to be for the CODE search results, not for the REPO search results 😉.